src/exports.jl

```diff
@@ -1118,12 +1125,12 @@ export minimal_block_reps
 export minimal_faces
 export minimal_generating_set
 export minimal_generator
-export minimal_size_generating_set, has_minimal_size_generating_set, set_minimal_size_generating_set
-export minimal_supercone_coordinates, minimal_supercone_indices, is_minimal_supercone_coordinate_vector
 export minimal_nonfaces
 export minimal_normal_subgroups, has_minimal_normal_subgroups, set_minimal_normal_subgroups
 export minimal_primes
+export minimal_size_generating_set, has_minimal_size_generating_set, set_minimal_size_generating_set
```

Comment:
This might be a case for application of `is_known` (in the loop in PR #4507 )? If so, maybe not export this now, but use `is_known` in a follow-up PR? I mean for `has_minimal_size_generating_set` -> `is_known(::typeof(minimal_size_generating_set, ...)`.

Reply:
Just comes from sorting the exports. So I do not want to deal with this here.